core/includes/theme.inc

```diff
@@ -1940,6 +1940,13 @@ function template_preprocess_image(&$variables) {
   $path_or_uri = $variables['uri'] ? $variables['uri'] : $variables['path'];
   $variables['attributes']['src'] = file_create_url($path_or_uri);
 
+  if (isset($variables['timestamp'])) {
+    $variables['attributes']['src'] = $variables['attributes']['src'] . (strpos($variables['attributes']['src'], '?') !== FALSE ? '&' : '?') . backdrop_http_build_query(array('timestamp' => $variables['timestamp']));
+  }
+  else {
+    $variables['attributes']['src'] = file_create_url($path_or_uri);
```

Comment:
Hm... isn't that _else_ redundant? The same value gets set in line 1941.

Reply:
Was just looking at that.

Reply:
Fixed. Waiting on tests...